### ISSUE TITLE
cannot find protocol declaration for 'SurvicateDelegate'.

### ISSUE THREAD

**Issue by hamed-rezaee** [andre-paraense/survicate-flutter-sdk#1]
I faced this issue when I try to run an example app on iOS:

```
error: cannot find protocol declaration for 'SurvicateDelegate'
    @interface SwiftSurvicateFlutterSDKPlugin : NSObject <FlutterPlugin, SurvicateDelegate>
```

would you help me with this, please?

**Comment by andre-paraense** [comment]
Hi, @hamed-fs ! Surely I will help you :)

Whilst I try to reproduce your problem, can you please provide more info about your environment, such as the output of your `flutter doctor` command, for instance?

**Comment by hamed-rezaee** [comment]
thanks a lot @andre-paraense, here you are:

```
[✓] Flutter (Channel stable, 1.22.5, on Mac OS X 10.15.7 19H2 darwin-x64, locale
    en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
[!] Android Studio (version 4.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.52.1)
 
[✓] Connected device (1 available)   
```
I just update the `Survicate` version to `1.4.3` and run the example.

```
  s.dependency 'Survicate', '1.4.3'
  s.platform = :ios, '10.0'
```

**Comment by andre-paraense** [comment]
I have been able to reproduce the error. Also, I already have a hypothesis for the root cause, so the library might need a new release. I will keep you updated, thanks for raising this :)

**Comment by hamed-rezaee** [comment]
> I have been able to reproduce the error. Also, I already have a hypothesis for the root cause, so the library might need a new release. I will keep you updated, thanks for raising this :)

sure, thank you. 🙇🏼

**Comment by andre-paraense** [comment]
If for some reason you reach a solution before, you are most welcome to contribute with a pull request as well :)

**Comment by andre-paraense** [comment]
Solved by PR #3
